Frontend/ui-driver/src/Resource/Localizable/TA.purs

```diff
@@ -1560,6 +1560,17 @@ getTA stringKey =
         YOU_GOT_N_POINTS points -> "நீங்கள் "<> points <>" புள்ளிகள் பெற்றுள்ளீர்கள்"
         DISCOUNTED -> "தள்ளுபடி"
         YATRI_POINTS_TNC -> "[புள்ளிகள் விதிமுறைகள் மற்றும் நிபந்தனைகள்]"
+        HOTSPOTS -> "ஹாட்ஸ்பாட்கள்"
+        VERY_HIGH -> "மிக உயர்ந்தது"
+        HIGH -> "உயர்"
+        VERY_HIGH_DEMAND_AREA -> "மிக அதிக தேவை உள்ள பகுதி"
+        HIGH_DEMAND_AREA -> "அதிக தேவை உள்ள பகுதி"
+        MODERATE -> "மிதமான"
+        AVERAGE_DEMAND_AREA -> "சராசரி தேவை பகுதி"
+        THIS_AREA_IS_EXPERIENCING_AVERAGE_SEARCHES -> "இந்தப் பகுதி வாடிக்கையாளர்களிடமிருந்து சராசரியான தேடல்களை எதிர்கொள்கிறது"
+        THIS_AREA_IS_EXPERIENCING_VERY_HIGH_SEARCHES -> "இந்தப் பகுதி வாடிக்கையாளர்களிடமிருந்து அதிக எண்ணிக்கையிலான தேடல்களை எதிர்கொள்கிறது"
```

Comment:
can you replace this with `THIS_AREA_IS_EXPERIENCING_VERY_HIGH_SEARCHES -> "இந்தப் பகுதி வாடிக்கையாளர்களிடமிருந்து அதிக எண்ணிக்கையிலான தேடல்களை எதிர்கொள்கிறது"`

this 
THIS_AREA_IS_EXPERIENCING_VERY_HIGH_SEARCHES -> "இந்தப் பகுதி வாடிக்கையாளர்களிடமிருந்து மிக அதிக எண்ணிக்கையிலான தேடல்களை எதிர்கொள்கிறது"